libs/books/data-access/src/lib/+state/reading-list.effects.ts

```diff
@@ -27,8 +35,27 @@ export class ReadingListEffects implements OnInitEffects {
   addBook$ = createEffect(() =>
     this.actions$.pipe(
       ofType(ReadingListActions.addToReadingList),
-      concatMap(({ book }) =>
+      concatMap(({ book, enableUndo }) =>
         this.http.post('/api/reading-list', book).pipe(
+          tap(() => {
```

Comment:
Shouldn't need to use `tap(...)` here. Can you handle the undo case with a new effect rather than modify an existing one?